### PR TITLE
Add a debug option to render cards dynamically instead of using images

### PR DESCRIPTION
**Pull request by alicetouhou**
This will make it easier to test visual changes to the cards

### CODE REVIEW

File: common/config/debug-config.example.js [hunk 1]
```diff
@@ -20,4 +20,5 @@ export default {
 	autoEndTurn: false,
 	logAttackHistory: false,
 	disableRewardCards: false,
-}
+	showCardsAsHTML: false,
```
Comment (Lunarmagpie) [inline common/config/debug-config.example.js:23]: I don't think this name makes a lot of sense

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

File: common/config/debug-config.example.js (hunk 1 above)
Comment (alicetouhou) [reply common/config/debug-config.example.js:23]: What would you make it?

Comment (Tyrannicodin) [reply common/config/debug-config.example.js:23]: How about `dynamicCardRenders`?

Comment (alicetouhou) [reply common/config/debug-config.example.js:23]: i like `renderCardsDynamically`

Comment (Lunarmagpie) [reply common/config/debug-config.example.js:23]: That sounds good

**Review by Lunarmagpie — APPROVED** [review]

**Pull request merged**